### ISSUE TITLE
AS create error message

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#764]
In GitLab by @eferre1 on Dec 22, 2021, 16:35

Hi @dbeltrankyl, @wuruchi, when running the create command on an existing experiment, if the user has removed the `pkl` or `plot` folders then the command fails when trying to write the joblist or the plot of the experiment, but the message is just a critical unhandled error:

```
Plotting the jobs list...
Plotting...
[ERROR] Trace: 
 [CRITICAL] Unhandled error: If you see this message, please report it in Autosubmit's GitLab project
```
It seems to be similar at what happened in #725 for the proj folder.

Could it be possible to add some checks and print a more informative message for this cases? No hurries about this, knowing it I'm able to spot it when someone asks, it is just that is not the first time that happens and I guess it won't be the last.

Thanks.

FYI: @mcastril

**Comment by kinow** [comment]
In GitLab by @wuruchi on Dec 22, 2021, 16:52

Related to #763.

Hello @eferre1 

Thanks for reporting this issue, it needs a better exception message.

Since @dbeltrankyl is going on vacation, I will work on this problem.

**Comment by kinow** [comment]
In GitLab by @wuruchi on Dec 27, 2021, 12:40

mentioned in commit e3e26a2de50f371ffa5519969bfd28ce2da399d9

**Comment by kinow** [comment]
In GitLab by @wuruchi on Dec 27, 2021, 12:47

mentioned in commit 2b4e049bceb48f043128de11cf992e33590b0149

**Comment by kinow** [comment]
In GitLab by @wuruchi on Dec 27, 2021, 12:51

mentioned in commit bc6ee22bce2e70d4098a5c70e346626b86f41f44

**Comment by kinow** [comment]
In GitLab by @wuruchi on Dec 27, 2021, 12:53

Hello @eferre1 

The fixes have been applied to 3.14 and 3.13; however, it will take until the next release of 3.13 for it to take effect.

**Comment by kinow** [comment]
In GitLab by @wuruchi on Jan 18, 2022, 12:31

Closing it. The fixes will take effect in the next release, which is expected very soon.
